docs/community/styleguide/formatting.md

```diff
@@ -0,0 +1,87 @@
+- Use short paragraphs or bullet points to convey the information.
+- Keep the language simple and direct.
+- Use an `>` in markdown to indicate the nature of the note or warning. 
+- Use the following syntax to apply a style. Currently our documenation supports **Remember** `<Remember>`:
+  * Surround the text you want to style with an opening <Remember> tag and a closing </Remember> tag.
+  * Note that the word 'Remember' does not need to be included within the tags, as it automatically provides the necessary styling.
+  * Use the following syntax to apply a style:
+  ` <Remember> 
+  No need to add a prefix; the tag automatically provides one
+  </Remember>`
+  
+  The output: 
+  <Remember> 
+  No need to add a prefix; the tag automatically provides one
+  </Remember>
+
+## Code blocks
+
+Code blocks are used to display code examples or snippets. 
+
+- In a Markdown document, use the `<CodeBlock>` tag and specify the language.
+  * Use the following syntax to apply a codeblock:
+  `<CodeBlock language="bash">
+  {`npm start`}
+  </CodeBlock>`
+  
+  The output:
+  
+  <CodeBlock language="bash">
+  {`npm start`}
+  </CodeBlock>
+  
+- Use code style for filenames, directories, and paths. For example: Go to the `/docs/tutorials` directory.
+- Choose a consistent number of spaces for indentation, such as 2 or 4 spaces, and use it consistently throughout the document.
+- Indent the code properly to show the structure and hierarchy of the code. Each level of indentation should align with the appropriate scope.
+- Avoid using tabs for indentation, as they may not render consistently across different platforms or text editors.
+For example, when writing code in Markdown, use four spaces for each level of indentation:
+```
+function myFunction() {
+    if (condition) {
+        console.log("Condition is true.");
+    } else {
+        console.log("Condition is false.");
+    }
+}
+```
+- Use single backticks to enclose inline code. For example, `asyncapi new --example=tutorial.yml --no-tty`
+- Remove any trailing spaces in the code. Trailing spaces can disrupt the readability and formatting of the code, so ensure they are removed.
+- Use triple backticks to enclose YAML code blocks. Specify the language as "yaml" within the backticks. This syntax is specifically for displaying code blocks that contain YAML content.
+  * Use this syntax:
+  ` ```yaml
+  asyncapi: '2.5.0'
+  info:
+  title: Account Service
+  version: 1.0.0
+  ``` `
+  
+  * The output:
+  ```yaml
+  asyncapi: '2.5.0'
+  info:
+  title: Account Service
+  version: 1.0.0
+  ```
+## Spacing
+
+Line spacing, or the amount of vertical space between lines of text in a paragraph, can aid or hinder reading. Adequate line spacing aids readers in navigating from the end of one line to the start of the next.
+
+- Leave a blank line between paragraphs to visually separate them. This helps readers distinguish between different sections of content.
```

Comment:
Specify whether a blank line should be left before or after headings and subheadings.